### DISCUSSION TITLE
Can m6anet pick up m1A signal?

### DISCUSSION THREAD

**Discussion by ruicatxiao** [GoekeLab/m6anet#94] [General]
Can the tool be able to able to tell apart m1A versus m6A? We are seeing in our organism some interesting signal from m6Anet inference that enriches at 5 prime end of transcripts. Similarly, our rRNA transcripts also showed strong inferred signal at 5 prime end.  Greatly appreciate your help!

**Comment by chrishendra93** [comment]
hi @ruicatxiao , we are not sure if m6Anet can tell apart m1A versus m6A. We tried to limit the detection to m6A by only focusing on the DRACH motif, but I suppose if a particular DRACH motif has an overlap with a motif that might contain m1A, the prediction might be affected. Do you think that some of the sites detected by m6A here are from m1A? 

Also moving this to the discussion since it is more appropriate

**Reply by ruicatxiao** [reply]
We found a few sites on rRNA that has strong inferred m6A level. Our organism has TRMT61a and METTL5, no other adenine MTase. This is what prompt us to think whether m6anet can detect m1A signal.
